FuzzTesting/Sources/FuzzBinary/main.swift

```diff
@@ -2,6 +2,8 @@ import Foundation
 
 import FuzzCommon
 
+import SwiftProtobuf
```

Comment:
So these could have been FoundationCompat? But no matter which one picks, to maintain Data support folks will have to explicitly import atleast one of them?

Reply:
They could have, yes. The codegen was changed to import `SwiftProtobufCore` in the autogenerated files instead of `SwiftProtobuf`, so unless `SwiftProtobuf` or `SwiftProtobufFoundationCompat` are also imported, the `Data` methods won't be included.

An alternative would be to re-export `SwiftProtobufFoundationCompat` in `FuzzCommon`, that would avoid the import here.

Reply:
> An alternative would be to re-export `SwiftProtobufFoundationCompat` in `FuzzCommon`, that would avoid the import here.

Ack.

I was mainly asking to think about the general impacts as these CLs are being done.  If we do go down the path some some generation option for Foundation support, we could change what `import` is generated.

Anyway, not expecting a change because of this, was just asking to make sure I'm reasoning correctly on why the files did have to change.